### PR TITLE
Add BuildCraft Item Pipe support

### PR DESCRIPTION
**Pull request by davboecki**

### CODE REVIEW

**Comment by dblsaiko** [comment]
Looks good!

**Comment by AlexIIL** [comment]
Just to check: do you need the logistic nbt tag sent to the client? Currently BC has a cache system that (currently) maps from ItemStacks to int ids as an optimisation for pipes, and I can make it remove this tag if it's not needed on the client (but of course it will stay on the server)

**Comment by davboecki** [comment]
OK so generally the information details are not needed clientside. The only thing that will probably be needed at some point is the information that an item has that information. We used to render our transport box around items from LP when they were moving inside BC pipes. We already do that for TD pipes again, and I would like to add that back in for LP pipes at some point. But for that I don't need the details, just that it is there. I think the advantage would be that your optimisation would probably be better again than assigning a new id to every LP item. 

TL;DR: I don't need the details, just some flag, that those information exist.
